src/java/picard/sam/revertsam/OutputMap.java

```diff
@@ -0,0 +1,82 @@
+package picard.sam.revertsam;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMReadGroupRecord;
+import picard.PicardException;
+import picard.util.TabbedTextFileWithHeaderParser;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents the mapping from each read group id to the corresponding output file
+ * that RevertSam is expected to write to.
+ */
+public class OutputMap {
```

Comment:
Why is this class public by the constructors package protected?  Shouldn't the class be package protected?